Dockerfile

```diff
@@ -6,6 +6,8 @@ ENV NODE_ENV=production
 ENV ELASTIC_APM_JS_BASE_SERVICE_NAME=opbeans-react
 RUN npm install
 
+RUN npm install pm2 -g
+
 COPY --from=opbeans/opbeans-frontend:latest /app/ /app/client/
 
-CMD ["npm", "start"]
+CMD ["pm2-runtime", "processes.config.js"]
```

Comment:
This will also start the workload script which was previously only run when started when initialized via the [apm-integration-testing](https://github.com/elastic/apm-integration-testing) repo. I'm not saying it's wrong to do it here also, but I'm not sure. We'd have to think about the use-cases for this Docker image 🤔

Reply:
Yeah, not sure what the best approach would be here. We could have a separate Dockerfile and pm2 config file for the app without the workload script, maybe? 🤔

Reply:
Suggestion:

1. Have two config files as you say:
  - `ecosystem.config.js` - This one doesn't include the workload script
  - `ecosystem-workload.config.js` - This one does include the workload script
2. The `Dockerfile` in this repo uses `ecosystem.config.js`, while the `Dockerfile` in the integration testing repo uses the other

Reply:
fixed.